region/maridia/inner-yellow.json

```diff
@@ -858,32 +904,22 @@
                     "Morph",
                     "Gravity",
                     "canTrickyJump",
-                    {"or":[
-                      "HiJump",
-                      "h_canFly",
-                      "canWalljump",
-                      "canSpringBallJumpMidAir",
-                      "canTrickyDashJump",
-                      "canTrickyUseFrozenEnemies"
-                    ]}
+                    {"enemyKill":{ "enemies": [ [ "Menu", "Menu" ] ] }}
                   ],
                   "note": [
                     "With Gravity, it's possible to move forward and mid-air morph into the morph passage.",
                     "The momentum can push Samus into the tunnel, no need to press forward while in mid-air."
-                  ]
+                  ],
+                  "devNote": "The obstacle is not needed because the enemies can be freely killed with canTrickyJump and Power Beam"
                 },
                 {
                   "name": "Maridia Bug Room Suitless MidAir Morph",
                   "notable": false,
                   "requires": [
                     "h_canTwoTileJumpMorph",
                     {"or":[
-                      "HiJump",
-                      "h_canFly",
-                      "canWalljump",
-                      "canSpringBallJumpMidAir",
-                      "canTrickyDashJump",
-                      "canTrickyUseFrozenEnemies"
+                      {"enemyDamage": { "enemy": "Menu", "type": "contact", "hits": 1 }},
```

Comment:
I think this should require the obstacle clear.
Meaning, put `enemyDamage` as a way to clear the obstacle.